README.md

```diff
@@ -102,6 +102,10 @@ The following variable defaults are defined in `defaults/main.yml`.
 
     list of IP addresses to ban (default: `[]`)
 
+* `minecraft_server_properties`
+
+    list of server.properties entries (e.g. `server-port=25565`) to set (default: `[]`)
```

Comment:
Actually, nevermind. :) I can just make it pass the `minecraft_server_properties` dict as a one-item list to `ServerProperties`. A little hacky, but it works, and it's just implementation that's hidden from the user.